### DISCUSSION TITLE
How can resolve ? : WARNING: Minimum density,    0.120 kg/m3, clipped in Mesh 1

### DISCUSSION THREAD

**Discussion by AntonioWeplan** [firemodels/fds#13098] [General]
Good morning,
WARNING: Minimum density,    0.120 kg/m3, clipped in Mesh 1

appears after 9 secondos of simulation.
How can I find the problems ?
I attached the file TXT FDS

[Scenario_B_Aprile_2024-Default - Copia.txt](https://github.com/user-attachments/files/15982127/Scenario_B_Aprile_2024-Default.-.Copia.txt)

**Comment by mcgratta** [comment]
I will take a look at your case.

**Comment by mcgratta** [comment]
![image](https://github.com/firemodels/fds/assets/11333911/37ec7b6d-f8ca-4a7b-9d99-5820d603ca92)

The problem lies at the base of the hollow column shown in the picture. You can see this by loading the Plot3D file that is generated when the numerical instability occurs. The WARNINGs that you are seeing indicates that there is a problem that in this case led to an instability. What are you trying to do within this column?

**Comment by AntonioWeplan** [comment]
Really thanks mcgratta for try to helpo me, after I wrote here, I saw the problem at the base of the hollow with "Maximum Errors in FDS Viewer", and I changed the shape. And now the simulation don't give me the error WARNING: Minimun density......
Now, the simulation interropt at 400 s and the message is: ERROR: Numerical Instability - FDS stopped (CHID: Scenario_B_Aprile_2024-Default)

I try to understand with the Maximum errors....but give me the red square on the material with burner.....maybe they are not in the same shape of the mesh? It's really hard to understand the error, really thanks for your help!

![Immagine 2024-06-27 154734](https://github.com/firemodels/fds/assets/173870010/956bb87d-2ace-4311-923d-a6b6ceccd049)

[2024.06.27 - Scenario_B_Aprile_2024-.txt](https://github.com/user-attachments/files/16015049/2024.06.27.-.Scenario_B_Aprile_2024-.txt)

**Comment by mcgratta** [comment]
Read in the Plot3D file and look for where the velocity vectors do not look physically correct. That is a  hint as to where the problem lies.

**Comment by AntonioWeplan** [comment]
Really really thanks mcgratta, the last request, How can I visualize a your image aboce the Plot 3d in vectors? can you give me the path of the command? thanks

**Comment by mcgratta** [comment]
Hit the v key.
